src/chapters/introduction/why-learn-python.rst

```diff
@@ -5,8 +5,9 @@ Most programmers have a favorite coding language, but there is NOT a single,
 best choice. Each language does some things very well, other things pretty well, and
 a few things not so well.
 
-For most tasks, any programming language will do the job. Which one you use
-depends on convenience and personal preference. So why start with Python?
+For most tasks, any programming language can do the job.
+You have to decide which one to use for a project based on what the language does very well and which languages you enjoy using.
```

Comment:
I had trouble following the flow of this sentence - too many which/what.

Sorry, but for this one I've got a full change suggestion:
"To pick which one to use, think about what each language does best and which languages you enjoy."

Reply:
Done!